sys/include/hashes/psa/riot_hashes.h

```diff
@@ -77,6 +77,13 @@ typedef sha512_256_context_t psa_hashes_sha512_256_ctx_t;
 #include "hashes/sha256.h"
 #endif
 
+#if (IS_USED(MODULE_PSA_RIOT_HASHES_SHA_3))
+#include "hashes/sha3.h"
+
+typedef keccak_state_t psa_hashes_sha3_ctx_t;
+#include "../psa_crypto/include/psa_hashes.h"
```

Comment:
Why is this needed for SHA-3 and not for the other hash software implementations?

Reply:
Thanks for the notice! The include is now happening in crypto_includes.h, as for the other hash software implementations too :)